### PR TITLE
made changes

### PR DESCRIPTION
**Pull request by sldless**

### CODE REVIEW

**Comment by Seniatical** [comment]
make sure to remove the "colour" kwarg from the payload

**Review by sldless — COMMENTED** [review]
a

**Comment by sldless** [comment]
a

File: acord/client/client.py [hunk 1]
```diff
@@ -286,7 +286,7 @@ async def change_presence(self, presence: Presence) -> None:
         await self.http.ws.send_str(payload.json())
 
     def run(self, token: str = None, *, reconnect: bool = True, resumed: bool = False):
-        """Runs the client, loop blocking
+        """Starts the client with discord api, With loop blocking.
```
Comment (Seniatical) [inline acord/client/client.py:289]: this don't make sense ._.

File: acord/client/client.py [hunk 2]
```diff
@@ -434,4 +432,6 @@ async def gof_channel(self, channel_id: int) -> Optional[Any]:
     
     @property
     def guilds(self):
+        """Returns guilds data, Only if the client is in that guild"""""
```
Comment (Seniatical) [inline acord/client/client.py:435]: What's with all the quotes

**Review by sldless — COMMENTED** [review]
A

**Pull request merged**